app/views/state_file/questions/initiate_data_transfer/edit.html.erb

```diff
@@ -4,15 +4,16 @@
   <h1 class="h2" id="main-question"><%= title %></h1>
   <p><%= t('.description') %></p>
 
-  <%= link_to @link.to_s, class: "button button--primary button--icon", id: "firstCta" do %>
-    <%= image_tag("icons/link-white.svg", alt: "") %>
-
-    <%= t(".button") %>
-  <% end %>
+  <% unless Rails.env.production? %>
+    <%= link_to @link.to_s, class: "button button--primary button--icon", id: "firstCta" do %>
+      <%= image_tag("icons/link-white.svg", alt: "") %>
+      <%= t(".button") %>
+    <% end %>
 
-  <% if ENV["IRS_TRANSFER_AUTH_URL"].present? %>
-    <%= link_to @irs_testing_link.to_s, class: "button", id: "firstCta" do %>
-      Transfer Data from IRS testing site (Don't click this unless you've been asked to do so)
+    <% if ENV["IRS_TRANSFER_AUTH_URL"].present? %>
+      <%= link_to @irs_testing_link.to_s, class: "button button--full-width text--small", id: "firstCta" do %>
```

Comment:
Is this what you were bringing up in our group discussion earlier? It sounds like we want this button to be fixed up to be nicer and work for either testing or production depending on the environment's credentials.

Reply:
@mpidcock yes going to remove it and leave styling changes for here https://www.pivotaltracker.com/story/show/186918612